Elcano_C2_LowLevel/Elcano_C2_LowLevel.ino

```diff
@@ -259,46 +259,53 @@ void setup()
 }
 /*---------------------------------------------------------------------------------------*/
 void loop() {
-  Serial.println("Start of loop");
+  //Serial.println("Start of loop");
   SerialData Results;
 
   // Save start time for performance report.
   unsigned long startTime = millis();
   // Get the next loop start time.
   unsigned long nextTime = startTime + LOOP_TIME_MS;
 
+  digitalWrite(27, LOW);
+  digitalWrite(33, LOW);
   startCapturingRCState();
 
   unsigned long local_results[7];
   //  PrintDone();
 
   while (millis() < nextTime &&
-         ~((RC_Done[RC_ESTP] == 1) && (RC_Done[RC_GO] == 1) && (RC_Done[RC_TURN] == 1) && (RC_Done[RC_RDR])))
+         ~((RC_Done[RC_ESTP] == 1) && (RC_Done[RC_GO] == 1) && (RC_Done[RC_TURN] == 1) && (RC_Done[RC_RDR] == 1)))
+  {
```

Comment:
Just FYI, this wasn't needed -- the semi-colon is a statement, so the while loop does have a body.  We're moving away from control statements without { } though, so it's ok to put them in.  For contrast, the following change was a bug fix -- look for old line 486 (new line 444) and following in the Elcano_C2_LowLevel.ino diff.  There, the body of the control statement had been commented out, so the control statement was controlling the following statement inadvertently.  I griped about this on Slack at some length back in July and we agreed to use { } around a single statement.

https://github.com/elcano/elcano/commit/572bcbc70352e5d1c16ff714abf31b11041c1242